src/courses/delta/02.md

```diff
@@ -0,0 +1,30 @@
+---
+order: 02
+next: 03.md
+title: 2. Delta Process Requirements
+author: Daniel Medina, George Dias
+---
+
+## 2.1 Requirements
+
+To conduct the delta process, the following items are needed:
+
+1. An InSpec profile:
+    a. Clone or download an existing profile from a repository source to your local environment.
+    b. See [What is an InSpec Profile](../beginner/02.md#what-is-an-inspec-profile) for additional information.
+2. A guidance file (XCCDF):
+    a. Download either a DISA STIG or a CIS Benchmark from the appropriate site.
+
+::: note Using Generated Profile Stubs
```

Comment:
```suggestion
::: details What if there is no existing InSpec profile?
```

I am assuming you are suggesting this information for an update like Windows 2016 -> Windows 2022 if, for example Windows 2022 did not yet have an InSpec profile? If my assumption is correct, I recommend putting it in a drop down (::: details) rather than (::: note) so people only have to expand and think about it if it is relevant to them.

Reply:
Agreed. Also framing it as a question the reader would have makes sense.